website/docs/about/index.md

```diff
@@ -18,58 +18,55 @@ Because **DOSBox Staging** comes with sensible defaults, you'll need to write
+- Faithfully emulate the **DOS operating environment** running on IBM PC
+  compatibles and the IBM PCjr with the primary goal of running **all PC
+  Booter, DOS and Windows 3.x games** released in the 1981--2000 period. Running
+  applications and demoscene productions or more recent DOS software is a
+  secondary objective.
+- Improve the **out-of-the-box experience**.
+- Focus on supporting up-to-date, **current operating systems** and **modern
+ hardware**.
+- Implement **new features** and **quality-of-life** improvements.
+- Deliver a consistent **cross-platform experience**.
+- Leverage **upstream** and **community developments** in DOSBox.
 - Encourage **new contributors** by removing barriers to entry.
-- Implement **new features** and quality-of-life improvements.
-- Fix, clean up, and integrate notable **community-developed patches** that
-  fill valuable general-purpose and/or gaming-focused emulation holes
-  (provided they don't impact broader compatibility, performance, stability,
-  or code maintainability).
-- Prioritize the **quality** of newly written code to minimize technical
-  debt and ease future maintenance, which generally means following the
-  [Staging Coding Style
-  Guide](https://github.com/dosbox-staging/dosbox-staging/blob/main/CONTRIBUTING.md#coding-style)
-  and being aware of best practices, like the [C++ Core
-  Guidelines](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines).
-- Deliver a consistent cross-platform experience.
-- Leverage upstream and community developments in DOSBox.
-- Focus on supporting up-to-date, current operating systems and modern
- hardware.
-
-Please refer to the [Feature highlights](../index.md#feature-highlights) section of our
-front page to learn about the advanced features DOSBox Staging offers.
-
-The key features for developers are summarised
-[here](https://github.com/dosbox-staging/dosbox-staging?tab=readme-ov-file#key-features-for-developers).
-
-We plan to add the features listed in [the
-backlog](https://github.com/dosbox-staging/dosbox-staging/projects/3).
+- Fix, clean up, and integrate notable **community-developed patches**.
+- Prioritise **code quality** to minimise technical
+  debt and ease maintenance. This generally means following the
+  [Staging Coding Style Guide](https://github.com/dosbox-staging/dosbox-staging/blob/main/CONTRIBUTING.md#coding-style)
+  and best practices such as the [C++ Core Guidelines](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines).
 
 
 ## Non-goals
 
-- Support old operating systems (Windows 9x/Me/XP, OS/2, and Mac OS X 10.5)
-  and limited CPU/memory hardware, which are constraints
-  [DOSBox](https://www.dosbox.com/) continues to support.
+- Support old, end-of-life operating systems (e.g., Windows 7 or older, OS/2,
+  or Mac OS X 10.5 or older) and limited CPU/memory hardware, which are
+  constraints the original [DOSBox](https://www.dosbox.com/) continues to
+  support.
 
```

Comment:
do we want to point to anything well-defined, like https://endoflife.date/windows, or leave it more at the team's discretion?

Reply:
I'd say the latter.